spec/ttml1.xml

```diff
@@ -5045,6 +5045,17 @@ then a presentation processor must use the closest supported value.</p>
 the computed padding and the supported padding is minimized. If there are multiple closest supported values equally distant from
 the computed value, then the value most distant from 0, i.e., the greatest padding, is used.</p>
 </note>
+<note role="elaboration">
+	<p>Padding space is applied as inset space to the part of the region's area that corresponds to the large allocation rectangle 
+	in <bibref ref="xsl11"/>. Because the extent of a TTML region defines the fixed dimensions of the large allocation rectangle, 
+	padding effectively reduces the available space for a content rectangle in the XSL-FO model. 
+	This needs to be taken into account when the region's extent is conceptually mapped to dimensions of a content rectangle. For this 
+	mapping the sum of the padding-before value and padding-after value and the sum of padding-start value and padding-end value need 
+	to be subtracted from the region's width and height (subject to the value of the region's writingMode).
+	To get the top left vertex of the content rectangle the values for padding-start and padding-before need to be added the region's 
+	origin (subject to the value of the region's writingMode).
+	</p>		
+</note>
```

Comment:
I think the key is to carefully define what _region area_ means since there is no defined XSL equivalent. For instance, `tts:overflow` attribute _defines whether a **region area** is clipped or not if the descendant areas of the region overflow its extent_, but I assume that the clipping occurs outside the _inset area_, i.e. the area resulting from the region inset by padding, and not outside the _region area_.